rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -270,6 +270,111 @@ def __init__(self,
 
+class ROCALAudioIterator(object):
+    """
+    ROCAL iterator for audio tasks for PyTorch
+    Please keep in mind that Tensors returned by the iterator are
+    still owned by ROCAL. They are valid till the next iterator call.
+    If the content needs to be preserved please copy it to another tensor.
+    Parameters
+    ----------
+    pipelines : list of amd.rocalLI.pipeline.Pipeline
+                List of pipelines to use
+    size : int
+           Number of samples in the epoch (Usually the size of the dataset).
+    auto_reset : bool, optional, default = False
+                 Whether the iterator resets itself for the next epoch
+                 or it requires reset() to be called separately.
+    fill_last_batch : bool, optional, default = True
+                 Whether to fill the last batch with data up to 'self.batch_size'.
+                 The iterator would return the first integer multiple
+                 of self._num_gpus * self.batch_size entries which exceeds 'size'.
+                 Setting this flag to False will cause the iterator to return
+                 exactly 'size' entries.
+    dynamic_shape: bool, optional, default = False
+                 Whether the shape of the output of the RALI pipeline can
+                 change during execution. If True, the pytorch tensor will be resized accordingly
+                 if the shape of RALI returned tensors changes during execution.
+                 If False, the iterator will fail in case of change.
+    last_batch_padded : bool, optional, default = False
+                 Whether the last batch provided by RALI is padded with the last sample
+                 or it just wraps up. In the conjunction with `fill_last_batch` it tells
+                 if the iterator returning last batch with data only partially filled with
+                 data from the current epoch is dropping padding samples or samples from
+                 the next epoch. If set to False next epoch will end sooner as data from
+                 it was consumed but dropped. If set to True next epoch would be the
+                 same length as the first one.
+    Example
+    -------
+    With the data set [1,2,3,4,5,6,7] and the batch size 2:
+    fill_last_batch = False, last_batch_padded = True  -> last batch = [7], next iteration will return [1, 2]
+    fill_last_batch = False, last_batch_padded = False -> last batch = [7], next iteration will return [2, 3]
+    fill_last_batch = True, last_batch_padded = True   -> last batch = [7, 7], next iteration will return [1, 2]
+    fill_last_batch = True, last_batch_padded = False  -> last batch = [7, 1], next iteration will return [2, 3]
+
+    """
+    def __init__(self, pipeline, tensor_layout = types.NCHW, reverse_channels = False, multiplier = [1.0, 1.0, 1.0], offset = [0.0, 0.0, 0.0], tensor_dtype = types.FLOAT, size = -1, auto_reset = False, device = "cpu", device_id = 0):
```

Comment:
Why the default layout for audio iterator is set to NCHW?

Reply:
changed to NONE